src/index.html

```diff
@@ -28,5 +28,6 @@
   </script>
   <script async src='https://www.google-analytics.com/analytics.js'></script>
   <script src="https://ajax.googleapis.com/ajax/libs/hammerjs/2.0.8/hammer.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/web-animations/2.2.5/web-animations.min.js"></script>
```

Comment:
I'm not sure how that would work out if the request takes too long and Angular is bootstrapped by then.